### PR TITLE
Update Teams Calling Policy

### PR DESCRIPTION
**Pull request by akazachenka**
- Add new properties `PayAsYouGoSpendingLimits` and `PayAsYouGoSpendingUserLimit` for Pay-as-you-go calling spending limits feature.
- Split parameters on new lines in `New-CsTeamsCallingPolicy.md` for better readability in the same way as they are in `Set-CsTeamsCallingPolicy.md`.

### CODE REVIEW

**Comment by akazachenka** [comment]
@microsoft-github-policy-service agree company="Microsoft"

File: teams/teams-ps/teams/Set-CsTeamsCallingPolicy.md [hunk 1]
```diff
@@ -484,6 +486,38 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -PayAsYouGoSpendingUserLimit
+Determines the value of spending limit for a user making pay-as-you-go PSTN calls.
```
Comment (michbrown-png) [inline teams/teams-ps/teams/Set-CsTeamsCallingPolicy.md:506]: Please add details on what the possible values are

File: teams/teams-ps/teams/New-CsTeamsCallingPolicy.md [hunk 2]
```diff
@@ -500,6 +523,37 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -PayAsYouGoSpendingUserLimit
+Determines the value of spending limit for a user making pay-as-you-go PSTN calls.
```
Comment (michbrown-png) [inline teams/teams-ps/teams/New-CsTeamsCallingPolicy.md:543]: Please add details on what the possible values are

Comment (akazachenka) [reply teams/teams-ps/teams/New-CsTeamsCallingPolicy.md:565]: fixed

File: teams/teams-ps/teams/Set-CsTeamsCallingPolicy.md (hunk 1 above)
Comment (akazachenka) [reply teams/teams-ps/teams/Set-CsTeamsCallingPolicy.md:528]: fixed

**Comment by akazachenka** [comment]
Properties were renamed, this PR is no longer relevant

**Pull request closed without merging**